README-MAINTAINER.md

```diff
@@ -0,0 +1,32 @@
+# Maintainer Guide - PostHog Android SDK
+
+This document contains information specifically for maintainers of the PostHog Android SDK.
+
+## Rotating Sonatype User Token
```

Comment:
could be part of https://github.com/PostHog/posthog-android/blob/main/RELEASING.md
reason is here https://github.com/PostHog/posthog-ios/pull/398#discussion_r2556406602